src/components/SchemaNode.svelte

```diff
@@ -9,7 +9,9 @@
     <span
       class="text-gray-700">{parentFields.join('.')}{parentFields.length ? '.' : ''}</span><span>{node.name}</span>
   </p>
-  {#if node.description}
+ {#if node.description === 'The channel the application is being distributed on.'}
```

Comment:
Is this change related to the issue?

Reply:
Sorry, no, that was for the initial test I needed to do before getting assigned a task.